packages/blade-mcp/knowledgebase/patterns/Settings.md

```diff
@@ -0,0 +1,391 @@
+## Pattern Name
```

Comment:
propagate all the changes from story comments to here once done.

Reply:
missed this .. 😢